pyproject.toml

```diff
@@ -1,89 +1,107 @@
 [project]
-name = 'drizzlepac'
-description = """ HST image combination using the drizzle algorithm to combine astronomical images, to model image distortion, to remove cosmic rays, and generally to improve the fidelity of data in the final image. """
-readme = { file = 'README.md', content-type = 'text/x-rst' }
-requires-python = '>=3.10'
-license = { file = "LICENSE.txt" }
-authors = [{ name = 'Megan Sosey' }, { name = 'Warren Hack' }, { name = 'Christopher Hanley' }, { name = 'Chris Sontag' }, { name = 'Mihai Cara' }, { name = 'Michael Dulude' }, { name = 'Michele De La Pena' }, { name = 'Steven Goldman' }]
+name = "drizzlepac"
+description = " HST image combination using the drizzle algorithm to combine astronomical images, to model image distortion, to remove cosmic rays, and generally to improve the fidelity of data in the final image. "
+requires-python = ">=3.10"
+authors = [
+    { name = "Megan Sosey" },
+    { name = "Warren Hack" },
+    { name = "Christopher Hanley" },
+    { name = "Chris Sontag" },
+    { name = "Mihai Cara" },
+    { name = "Michael Dulude" },
+    { name = "Michele De La Pena" },
+    { name = "Steven Goldman" },
+]
 classifiers = [
-    'Intended Audience :: Science/Research',
+    "Intended Audience :: Science/Research",
     "Topic :: Scientific/Engineering :: Astronomy",
-    'License :: OSI Approved :: BSD License',
-    'Operating System :: MacOS :: MacOS X',
-    'Operating System :: POSIX',
-    'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3 :: Only',
-    'Programming Language :: Python :: 3.8',
-    'Programming Language :: Python :: 3.9'
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
 ]
 dependencies = [
-    'astropy>=5.0.4',
-    'fitsblender>=0.4.2',
-    'scipy',
-    'matplotlib',
-    'stsci.tools>=4.0',
-    'stsci.image>=2.3.7',
-    'stsci.imagestats',
-    'stsci.skypac>=1.0.9',
-    'stsci.stimage',
-    'stwcs>=1.5.3',
-    'tweakwcs>=0.8.2',
-    'stregion>=1.1.7',
-    'requests',
-    'scikit-learn>=0.20',
-    'bokeh',
-    'pandas',
-    'spherical_geometry>=1.2.22',
-    'astroquery>=0.4',
-    'astrocut<=0.9',
-    'photutils>1.5.0',
-    'lxml',
-    'PyPDF2',
-    'scikit-image>=0.14.2',
-    'tables',
-    'typing_extensions>4.0',
-    'markupsafe<=2.0.1',
+    "astropy>=5.0.4",
+    "fitsblender>=0.4.2",
+    "scipy",
+    "matplotlib",
+    "stsci.tools>=4.0",
+    "stsci.image>=2.3.7",
+    "stsci.imagestats",
```

Comment:
We want to make sure drizzlepac uses stsci.imagestats >= 1.8.2 (**_NOT_** 1.7.0, 1.8.0, or 1.81. though I see these are currently "yanked").